protocols/direct-fund/direct-fund.go

```diff
@@ -1,34 +1,35 @@
-package protocols
+package directfund
 
 import (
 	"errors"
 	"fmt"
 	"math/big"
 
 	"github.com/statechannels/go-nitro/channel/state"
+	"github.com/statechannels/go-nitro/protocols"
```

Comment:
This "suggestion" isn't really a suggestion - I don't think it's better than the current version - but I just wanted to point it out.

The dot import brings the imported package's identifiers into global namespace, so that, eg, `protocols.SideEffects{}` can just be `SideEffects{}`. These packages (`protocols` and its children) are about as good a case for the dot import as I can think of, but still it brings threat of namespace collisions which I'd rather not have to think about.

```suggestion
	. "github.com/statechannels/go-nitro/protocols"
```